transfer/metadata.md

```diff
@@ -2,141 +2,6 @@
-
-Possible to have a more robust data model; look at AS model. Is this desirable?
-
-*   role (could be something other than creator)
-*   type (person/organization)
-*   URI
-*   Name String
-
-## Record type
-
-*   **Definition**: The broad category into which the records fall.
-*   **Purpose**: Allows different validation rules for transfer metadata to be used for different kinds of records.
-*   **Data type**: Locally controlled(?): administrative records, board materials, communications and publications, grant records.
-*   **Obligation**: Required
-*   **Repeatability**: Yes
-*   **Examples**:
-
-If multiple record types are allowed in a single transfer, how do we distinguish which is which? Does it matter? OTOH, making these transfers semantically meaningful would seem to indicate that we only have one type of record per transfer.
-
-## Languages
-
-*   **Definition**: The natural language(s) in which the materials are written.
-*   **Purpose**: Assists user evaluation of relevance and facilitates machine translation.
-*   **Data type**: Values must meet ISO 639-2: Codes for the Representation of Names of Languages.
-*   **Obligation**: Required
-*   **Repeatability**: Yes
-*   **Examples**:
-
-Use URIs? What about records that have no language content?
-
-Could also make this non-repeatable and use `mul` if there are multiple languages present.
-
-## Description
-
-*   **Definition**: A prose description of the nature and contents of the group of records.
-*   **Purpose**: Aids user discovery by providing description about the content and context of records.
-*   **Data type**: String
-*   **Obligation**: Required
-*   **Repeatability**: No
-*   **Examples**:
-
-## Restrictions
-
-*   **Definition**: Any specific restrictions related specifically to the group of records. Do not record general restrictions related to all records (or record types) from a given person or organization.
-*   **Purpose**: Provides information to users about the availability of records, and assists archivists in responsibly providing access to records.
-*   **Data type**: String
-*   **Obligation**: Optional
-*   **Repeatability**: No
-*   **Examples**:
-
-Depends a lot on what "restriction" means to our donors; may have to just be a text field.
-
-Wouldn't it be great if we had a set of locally-maintained donor-specific restrictions? Or would that be a waste of time?
-
-What about IP? Could point to [rightsstatements.org](http://rightsstatements.org/en/) for this...
-
-## Optional data elements
-
-*   **Definition**: Additional donor-supplied elements
-*   **Purpose**: Allows for the transfer of additional metadata elements which may be relevant to an understanding of the records.
-*   **Data type**: JSON key value pairs or arrays
-*   **Obligation**: Optional
-*   **Repeatability**: No
-*   **Examples**:
+The RAC will not index this metadata or include it in our discovery platform, but we will preserve it alongside the records that it pertains to.
```

Comment:
This file will be preserved as a bitstream alongside the records that it pertains to, but it may not be indexed as structured data.